src/utils/parsers.rs

```diff
@@ -0,0 +1,87 @@
+pub use teloxide_macros::TelegramCommandEnum;
+/// Enum for telegram commands
+///
+/// Example:
+/// ```
+/// use teloxide::utils::TelegramCommandEnum;
+/// use teloxide::utils::parse_command_into_enum;
+/// #[derive(TelegramCommandEnum, PartialEq, Debug)]
+/// enum TelegramCommand {
+///     Start,
+///     Help,
+/// }
+/// let (command, args) = parse_command_into_enum::<TelegramCommand>("/", "/start arg1 arg2").unwrap();
+/// assert_eq!(command, TelegramCommand::Start);
+/// assert_eq!(args, vec!["arg1", "arg2"]);
+/// ```
+pub trait TelegramCommandEnum: Sized {
+    fn try_from(s: &str) -> Option<Self>;
+}
+
+pub fn parse_command_into_enum<'a, T>(
+    prefix: &str,
+    text: &'a str,
+) -> Option<(T, Vec<&'a str>)>
+where
+    T: TelegramCommandEnum,
+{
+    let (command, args) = parse_command(prefix,text)?;
+    return match T::try_from(command) {
```

Comment:
Replace `return match ... {};` with just `match ... {}`? (For unified style)

Reply:
Probably RustFmt could fix that

Reply:
fixed